sdk/test-utils/test-utils-vitest/README.md

```diff
@@ -0,0 +1,120 @@
+follows:
+
+```bash
+cd /path/to/my/github/repositories
+git clone https://github.com/Azure/azure-sdk-for-js/
+```
+
+Having cloned this repository, let's set it up by running the following rush commands:
+
+```bash
+cd azure-sdk-for-js
+rush install
+rush build
+```
+
+This will optimistically assume you're in a fresh clone.
+
+From this point forward, we'll assume that you're developing (perhaps contributing!) to one of the
+`azure-sdk-for-js`'s libraries. So, your next step is to change directory to the path relevant to
+your project. Let's say you want to add the `@azure/test-utils-vitest` package to
+`@azure/keyvault-keys`, you'll be doing the following:
+
+```bash
+cd sdk/keyvault/keyvault-keys
+```
+
+Once there, you can add the `test-utils` package by changing your package.json
+to include the following line in the `devDependencies` section:
+
+```json
+{
+  // ... your package.json properties
+  "devDependencies": {
+    // ... your devDependencies
+    "@azure/test-utils-vitest": "^1.0.0",
+    // ... more of your devDependencies
+  },
+  // ... more of your package.json properties
+}
+```
+
+After that, we recommend you to update rush and install the dependencies again, as follows:
+
+```bash
+rush update
+```
+
+## Key concepts
+
+
+### Custom Testing Matrix
+
+Most Azure SDK for JavaScript libraries support multiple methods of authentication. The `@azure/test-utils-vitest` library attempts to add testing support for writing a single test suite then running your suite multiple times based on a provided testing matrix. The most common usage is to construct your client with a different authentication method for test runs.
+
+## Examples
+
+### Import functions
+
+```javascript
+import { matrix } from "@azure/test-utils-vitest";
+```
+
+### Wrap top-level test suite to run test matrix
+
+Wrap the top level `describe` of a test file to run the suite with the provided values.
+
+```typescript
+matrix(
+  [
+    [true, false],
+    [1, 2, 3]
+  ] as const,
+  (enabled: boolean, attempts: number) => {
+    describe(`Run with flag ${enabled ? "" : "not "}enabled and ${attempts} attempts`, () => {
+      // ...
+    });
+  }
+);
+```
+
+`matrix` takes a jagged 2D array and a function. It then runs this function with every possible combination of elements of each of the arrays. The example above will therefore generate 6 different test suites based on the values passed.
+
+## Troubleshooting
+
+Besides the usual debugging of your code and tests, if you ever encounter a problem, please follow
+up the [contributing](#contributing) guidelines on how to write an issue for us. We'll make sure to
+handle it as soon as we find the time.
+
+## Next steps
+
+Check out the [source folder](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/test-utils/src/) and the [test folder](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/test-utils/test/).
```

Comment:
this link should point to the new folder.

Reply:
Fixed